funnel/models/event.py

```diff
@@ -143,5 +202,32 @@ class SyncTicket(BaseMixin, db.Model):
     ticket_client_id = db.Column(db.Integer, db.ForeignKey('ticket_client.id'), nullable=False)
     ticket_client = db.relationship(TicketClient,
         backref=db.backref('sync_tickets', cascade='all, delete-orphan'))
-
     __table_args__ = (db.UniqueConstraint('proposal_space_id', 'order_no', 'ticket_no'),)
+
+    @classmethod
+    def get(cls, space, order_no, ticket_no):
+        return cls.query.filter_by(proposal_space=space, order_no=order_no, ticket_no=ticket_no).one_or_none()
```

Comment:
Are `order_no` and `ticket_no` guaranteed unique? Doesn't that depend on `ticket_client` as well? You'll have to accept `ticket_client` instead of `space` here and update the unique constraint above as well.

You may also want to remove `proposal_space` from this model as that is irrelevant if `ticket_client` also links to the space.